[docs] fix document date-time-functions typo

steadyBoy opened the pull request:
1. en: add convert_tz.md
2. en: fix curdate.md, the fix is to modify the title
3. en: add curtime.md
4. en: fix str_to_date.md, the fix is to modify the title
5. zh-CN: fix convert_tz.md, the fix is to modify the description
6. zh-CN: fix curdate.md, the fix is to modify the title

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (I Don't know)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

steadyBoy committed:
[docs] fix document date-time-functions typo
1. en: add convert_tz.md
2. en: fix curdate.md, the fix is to modify the title
3. en: add curtime.md
4. en: fix str_to_date.md, the fix is to modify the title
5. zh-CN: fix convert_tz.md, the fix is to modify the description
6. zh-CN: fix curdate.md, the fix is to modify the title

hf200012 reviewed (dismissed):
LGTM

qidaye commented on docs/en/sql-reference/sql-functions/aggregate-functions/group_concat.md:
```diff
@@ -0,0 +1,63 @@
+---
+{
+    "title": "group_concat",
```
`group_concat` already exists in string-functions subdirectory.

qidaye commented on docs/zh-CN/sql-reference/sql-functions/aggregate-functions/group_concat.md:
```diff
@@ -0,0 +1,63 @@
+---
+{
+    "title": "group_concat",
```
Same as above.

qidaye commented on docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md:
```diff
@@ -25,14 +25,12 @@ under the License.
 # convert_tz
-
+## description
 ## Syntax
```
```suggestion
### Syntax
```

steadyBoy committed:
[docs] fix document date-time-functions typo
1. Format modification

qidaye reviewed (dismissed):
LGTM

yangzhg commented on docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md:
```diff
@@ -25,14 +25,12 @@ under the License.
-
-## Syntax
+## description
+### Syntax
```
why？

steadyBoy replied:
Thanks for reminding. It has been modified

yangzhg reviewed (commented):
You should change `docs/.vuepress/sidebar` to make you rename visible

steadyBoy committed:
[docs] fix document date-time-functions typo
1. Change docs/.vuepress/sidebar
2. fix structure for docs/en/sql-reference/sql-functions/date-time-functions/curtime.md

qidaye commented on docs/en/sql-reference/sql-functions/date-time-functions/curdate.md:
```diff
@@ -24,7 +24,7 @@ specific language governing permissions and limitations
 -->
 
-# curdate
+# curdate,current_date
```
Add `current_date` to  keyword too.

qidaye commented on docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md:
```diff
@@ -24,7 +24,7 @@ specific language governing permissions and limitations
 -->
 
-# curdate
+# curdate,current_date
```
Same as above.

steadyBoy committed:
[docs] fix document date-time-functions typo
1. fix keyword for curdate.md

qidaye commented on docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md:
```diff
@@ -24,7 +24,7 @@ specific language governing permissions and limitations
 -->
 
-# curdate
+# curdate,`current_date`
```
```suggestion
# curdate,current_date
```

steadyBoy replied:
Thanks for reminding. It has been modified

steadyBoy committed:
[docs] fix document date-time-functions typo
1. fix keyword for curdate.md

steadyBoy committed:
[docs] fix document date-time-functions typo
1. fix keyword for curdate.md

steadyBoy committed:
[docs] fix document date-time-functions typo
fix structure for docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md

qidaye reviewed (dismissed):
LGTM

morningman commented:
hi @steadyBoy ,please rebase the master to solve the conflict.

qidaye commented:
@steadyBoy your rebase is not correct, please recheck.

qidaye reviewed (approved):
LGTM

morningman reviewed (approved):
LGTM
